test/functional/txn_clone.py

```diff
@@ -45,11 +45,11 @@ def run_test(self):
         # Coins are sent to node1_address
         node1_address = self.nodes[1].getnewaddress("from0")
 
-        # Send tx1, and another transaction tx2 that won't be cloned 
```

Comment:
This change line and a few others aren't supposed to happen in a scripted diff.

Reply:
Oops